pkg/kubecost/allocation.go

```diff
@@ -59,8 +59,10 @@ type Allocation struct {
 	CPUCoreRequestAverage  float64               `json:"cpuCoreRequestAverage"`
 	CPUCoreUsageAverage    float64               `json:"cpuCoreUsageAverage"`
 	CPUCost                float64               `json:"cpuCost"`
+	CPUAdjustment          float64               `json:"cpuAdjustment"`
```

Comment:
Can we call all of these adjustments `CostAdjustment`? e.g. `CPUCostAdjustment`, etc. It's not clear what's being adjusted as-written.